pkg/controller/performanceprofile/components/tuned/tuned.go

```diff
@@ -166,6 +168,21 @@ func NewNodePerformance(profile *performancev2.PerformanceProfile) (*tunedv1.Tun
 		}
 	}
 
+	// TODO: I think we need to parse additional kernel arguments and in case when it intersect with workload hints
+	// kernel arguments we should or override it or drop an error
+	// set default values for workload hints
```

Comment:
I tend to agree to the comment.

Reply:
I will provide it after as an improvement.

Reply:
fine for me